elf2flt.c

```diff
@@ -372,7 +372,7 @@ output_relocs (
 #endif
 
 #if 0
-  printf("%s(%d): output_relocs(abs_bfd=%d,symbols=0x%x,number_of_symbols=%d,"
+  printf("%s(%d): output_relocs(abs_bfd=%d,synbols=0x%x,number_of_symbols=%d"
```

Comment:
unfortunately it seems that we had some unrelated fixes in the original changes :(.  can we preserve these ?  maybe fix & squash into the reverts so the reverts are just partial ?

Reply:
I can certainly do that. A very good reason to not mix unrelated changes in a single commit.